app/Http/Controllers/Admin/RoleController.php

```diff
@@ -0,0 +1,337 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use Illuminate\Http\Request;
+use Yajra\Datatables\Datatables;
+use App\Http\Controllers\Controller;
+use App\Repositories\RoleRepository;
+use App\Repositories\PermissionRepository;
+use Validator;
+use Entrust;
+use DB;
+
+class RoleController extends Controller
+{
+    public $roleRepository;
+    public $permissionRepository;
+
+    /**
+     * Create a new controller instance.
+     *
+     * @return void
+     */
+    public function __construct(RoleRepository $roleRepository, PermissionRepository $permissionRepository)
+    {
+        $this->roleRepository = $roleRepository;
+        $this->permissionRepository = $permissionRepository;
+    }
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        return view('admin.roles.index');
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        //
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        $data = $request->all();
+
+        DB::beginTransaction();
+
+        try {
+            $rules = [
+                'name'          =>  'required|string|unique:roles',
+                'display_name'  =>  'required|string|unique:roles',
+            ];
+
+            $messages = [
+                'name.required'         =>  __('Role') . __('Required'),
+                'name.string'           =>  __('Role') . __('String'),
+                'name.unique'           =>  __('Role') . __('Unique'),
+                'display_name.required' =>  __('Display name') . __('Required'),
+                'display_name.string'   =>  __('Display name') . __('String'),
+                'display_name.unique'   =>  __('Display name') . __('Unique'),
+            ];
+
+            $validator = Validator::make($data, $rules, $messages);
+
+            if ($validator->fails()) {
+                return response()->json([
+                  'error'   =>  'valid',
+                  'message' =>  $validator->errors()
+                ]);
+            } else {
+                $this->roleRepository->create($data);
+
+                DB::commit();
+
+                return response()->json([
+                    'error'     =>  false,
+                    'message'   =>  'Success !'
+                ]);
+            }
+        } catch(Exception $e) {
+            return response()->json([
+                'error'     =>  true,
+                'message'   =>  'Fail !'
+            ]);
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.SpaceAfterDoubleArrow</br>**Severity**: error</br>**File**: app/Http/Controllers/Admin/RoleController.php</br>Expected 1 space between double arrow and "'Fail !'"; 2 found